site/app/views/calendar/CalendarView.php

```diff
@@ -103,6 +103,7 @@ public function showCalendar(CalendarInfo $info, array $courses, bool $in_course
             "color_options" => $course_colors,
             "show_all_cookie" => isset($_COOKIE['calendar_show_all']) ? $_COOKIE['calendar_show_all'] : 1,
             "calendar_course_cookie" => isset($_COOKIE['calendar_course']) ? $_COOKIE['calendar_course'] : $formatted_courses[0],
+            "display_name_cookie" => isset($_COOKIE['display_name']) ? $_COOKIE['display_name'] : '',
```

Comment:
You could use PHP's [null coalescing](https://www.php.net/manual/en/language.operators.comparison.php#language.operators.comparison.coalesce) operator here:

```suggestion
            "display_name_cookie" => $_COOKIE['display_name'] ?? '',
```